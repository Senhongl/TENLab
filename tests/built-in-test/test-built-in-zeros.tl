a = [2,2,3];
b = zeros(a);
print(b);
