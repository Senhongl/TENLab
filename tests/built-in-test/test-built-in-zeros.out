(1,.,.) = 
  0  0  0
  0  0  0

(2,.,.) = 
  0  0  0
  0  0  0
[ CPUIntType{2,2,3} ]